src/components/PostsList.tsx

```diff
@@ -1,85 +1,94 @@
 import React from 'react';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  posts: Post[],
+  selectPost: Post | null,
+  openerDetails: (post: Post) => void,
+  closeCommentList: React.Dispatch<React.SetStateAction<boolean>>,
```

Comment:
The type of 'closeCommentList' prop should be '() => void' but not 'React.Dispatch<React.SetStateAction<boolean>>'. It's better to handle the state outside of component.